ui/src/app/routing/workspace-app-routing.tsx

```diff
@@ -110,7 +110,7 @@ export const WorkspaceRoutes = () => {
     <AppRoute exact path={`${path}/terminals`}>
       <LeonardoAppRedirectPage
           routeData={{
-            breadcrumb: BreadcrumbType.Terminal,
+            breadcrumb: BreadcrumbType.Workspace,
```

Comment:
Unless I'm missing something, there are no terminal entities which make sense as breadcrumb elements.

We do still need breadcrumbs on the terminal page to navigate back to the workspace.